org.sil.reading-app-builder.yml

```diff
@@ -0,0 +1,158 @@
+# REFERENCES:
+# Python2.7:  https://github.com/flathub/shared-modules/blob/cc50bdbb34cc3d29bf61cfea5e27cd980b7bda04/python2.7/python-2.7.json
+# OpenJDK17: https://github.com/flathub/org.freedesktop.Sdk.Extension.openjdk17/blob/branch/23.08/org.freedesktop.Sdk.Extension.openjdk17.yaml
+# Node20:    https://github.com/flathub/org.freedesktop.Sdk.Extension.node20/blob/branch/23.08/org.freedesktop.Sdk.Extension.node20.yaml
+app-id: org.sil.reading-app-builder
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk17
+finish-args:
+  # Window x11 socket access
+  - --socket=x11
+  # Audio access
+  - --socket=pulseaudio
+  # Network access
+  - --share=network
+  # XShm access
+  - --share=ipc
+  # All external devices (catch all for devices we interact with and graphical acceleration)
+  - --device=all
+  # Local file browsing access
+  - --filesystem=host
+  # USB Drives
+  - --system-talk-name=org.freedesktop.Udisks2
+  # File saving access
+  - --talk-name=org.freedesktop.FileManager1
+
+# App's run command
+command: rab-run.sh
+
+modules:
+  - name: xterm
```

Comment:
why  do you need an xtern?

Reply:
We will be adding a way to build a Progressive Web App (PWA) output from the project data along with an Android app. That is not ready yet. I had removed the Node usage (https://github.com/flathub/flathub/pull/5629/commits/aaf816249051ec8056bf1c9aff6ddc1ed97a52d6) and I could also remove xterm for now. 

The purpose of using xterm is to allow `npm run preview` to run in a separate visible process so that the user can interact with it (get the local ip address / port to access the PWA in preview mode) and close the development web server running.

Should I remove it for now and then add it back in later when the feature is ready?

Reply:
if you don't need it it has no reason to be here.

Reply:
xterm has been removed.